source/includes/builders.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+// start-imports
+use MongoDB\Builder\Pipeline;
+use MongoDB\Builder\Query;
+use MongoDB\Builder\Stage;
+// end-imports
+
+require 'vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+// start-db-coll
+$collection = $client->sample_geospatial->shipwrecks;
+// end-db-coll
+
+// start-find
+// Creates a query filter by using builders and
+// retrieves matching documents
+$docs = $collection->find(Query::query(
+    feature_type: Query::eq('Wrecks - Visible'),
+    coordinates: Query::near(
+        Query::geometry(
+            type: 'Point',
+            coordinates: [-79.9, 9.3],
+        ),
```

Comment:
Do you think a method `Geometry::point(coordinates: [-79.9, 9.3])` would be useful for this?

Reply:
No, i think that users might appreciate this syntax mirroring the MQL so that they can use the server docs to get more context. So it might not be helpful to a large number of users to create this convenience method